content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -242,3 +242,4 @@ Ensure your {{< glossary_tooltip text="CNI" term_id="cni" >}} provider supports
 
 
 * [Validate IPv4/IPv6 dual-stack](/docs/tasks/network/validate-dual-stack) networking
+* [Dual-stack support with kubeadm](/docs/setup/production-environment/tools/kubeadm/dual-stack-support/)
```

Comment:
```suggestion
* [Enable dual-stack networking using kubeadm
](/docs/setup/production-environment/tools/kubeadm/dual-stack-support/)
```
(if making the suggested change to the title of that page)

However, this is just an example. Really I'd prefer to see “support” removed from the filename and generated URL.